src/plugins/discourse/weights.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import {type NodeWeight} from "../../core/weights";
+import {DEFAULT_TRUST_LEVEL_TO_WEIGHT} from "./createGraph";
+import {type User} from "./fetch";
+
+export function likeWeight(user: ?User): NodeWeight {
+  if (user == null) {
+    return 0;
+  }
+  return _trustLevelWeight(user);
+}
+
+export function _trustLevelWeight({trustLevel}: User): NodeWeight {
+  if (trustLevel == null) {
+    // The null trust level shouldn't happen in practice, right now users who
+    // only like but never post will have null trust level (will be fixed by #2045).
+    // This means they could have trust level 1. But to be conservative, we treat anyone
+    // with a null trust level as if they have trust level 0.
+    // Possibly this could come up with deleted users too.
+    return 0;
```

Comment:
Hmm, the comment and impl are inconsistent; it says we treat someone with null trust level as though they have TL0, but actually we return hardcoded 0, even if the weight for TL0 is something else (e.g. 0.1).

I know it's not a new bug in this PR, but maybe fix while you're touching this code? Or just add a comment, it would be hard to test w/o making the trust level dict customizable.